conf.d/flaskapp.conf

```diff
@@ -0,0 +1,14 @@
+server {
```

Comment:
Nit: It might be nice to name the folder something a little more intuitive. For instance, there's multiple services that look for a `conf.d` so naming this something like `nginx` would give a bit more context to someone viewing the repository for the first time.